packages/pwa/app/components/forms/credit-card-fields.jsx

```diff
@@ -106,6 +106,7 @@ const CreditCardFields = ({form, prefix = ''}) => {
                                     placement="top"
                                     label={formatMessage(
                                         {
+                                            id: 'tool_tip.info.cvv_code',
```

Comment:
At what point do we use something like `tool_tip` over using the component name like it's being used elsewhere?

Reply:
Thanks for pointing that out, after a short consideration, it makes sense to change it to `credit_card_fields.tool_tip.security_code` for a better context for this id